Test/Checkpoint-Migration.Tests.ps1

```diff
@@ -202,6 +202,9 @@ function Pop-Migration
     Add-Table -Name 'NotReplicated' -Column {
         int 'ID' -Identity -NotForReplication
     }
+'@ -Database ('RivetTest', 'RivetTest2')
+    ThenMigration -HasContent @'

```

Comment:
I notice this check is happening a few times. I think we need to check that not only are the `Add-Row` operations there, but that it exported more than one of the migration records. Rename `ThenMigration` to `ThenSchema` and add a `-ContainsRowsFor MIGRATION_NAME,MIGRATION_NAME2`. You'll want one test with at least two migrations to ensure all migrations are exported. You'll also want to check that Rivet's schema migrations aren't in the file.